monitoring/uss_qualifier/suites/faa/uft/design/README.md

```diff
@@ -0,0 +1,79 @@
+# UFT Message Signing tests for SCD using Uss_Qualifier
+
+UTM Field Test (UFT) has a requirement for USSes to sign their message requests
+and responses in the SCD flow using IATF certificates provided by FAA.
+
+This test suite helps to test the message signing by extending [suites.astm.utm.f3548_21](../../../astm/utm/f3548_21.yaml)
+with scenarios that trigger reporting of message signing in the interactions with mock_uss.
+
+## Test Harness Setup
+
+The test setup includes running the following components on your local machine, as also shown in the [diagram](./InterUss%20Test%20Harness%20With%20Message%20Signing.png)
+
+1. Uss Qualifier - This is the test driver that injects the operations for the SCD flow tests in USSes.
+2. Dummy Auth - This is a dummy auth server that provides tokens for uss-to-uss and uss-to-dss communication. Runs as a dockerized container at port 8085.
```

Comment:
I'd suggest that the auth source doesn't need to be Dummy Auth in the general case -- could we enable the general case while also describing the specific case by saying something like:

> 2. Auth - This auth server providers access tokens and the public key for token validation for USS-to-USS and USS-to-DSS communication.  In a local deployment of the test infrastructure, this can be supplied by an instance of dummy auth running as a dockerized container exposing port 8085, as per build/dev/run_locally.sh.

Same comment for DSS and Mock USS.